Content.Shared/_Sunrise/MentorHelp/SharedMentorHelpSystem.cs

```diff
@@ -273,4 +273,34 @@ public MentorHelpOpenTicketMessage(int ticketId)
             TicketId = ticketId;
         }
     }
+
+    [Serializable, NetSerializable]
+    public sealed class MentorHelpClientTypingUpdated : EntityEventArgs
+    {
+        public int TicketId { get; }
+        public bool Typing { get; }
+
+        public MentorHelpClientTypingUpdated(int ticketId, bool typing)
+        {
+            TicketId = ticketId;
+            Typing = typing;
+        }
+    }
```

Comment:
```suggestion
    public sealed class MentorHelpClientTypingUpdated(int ticketId, bool typing) : EntityEventArgs
    {
        public readonly int TicketId = ticketId;
        public readonly  bool Typing = typing;
    }
```

Во-первых такой упрощенный конструктор - стандарт.
Во-вторых геттеры в кодбазе писать не принято, лучше модиикатор `readonly`